plasmapy/particles/elements.py

```diff
@@ -1,14 +1,19 @@
 """
-Dictionaries containing basic atomic data.
+Module for loading elemental data from :file:`plasmapy/particles/data/elements.json`.
```

Comment:
Maybe let's just 
```suggestion
Module for loading data from :file:`plasmapy/particles/data/elements.json`.
```
because when I see this, I think of https://www.youtube.com/watch?v=lEANk0ItkX8

Reply:
By the way, do you think there's space in the open source community for reviews made purely through **angry water noises**?

Reply:
I hear a side project coming on! Can we also do a package of Tim Allen Home Improvement grunts? That's basically how I communicate with my computer screen all day.